### PR TITLE
[react-modal] Added id attribute, as per PR https://github.com/reactjs/react-modal/765

### PR DESCRIPTION
**Pull request by nitzanmo**
…pull/765

Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/reactjs/react-modal/765
- [X] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [X] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/react-modal/react-modal-tests.tsx [hunk 1]
```diff
@@ -64,6 +64,7 @@ class ExampleOfUsingReactModal extends React.Component {
     };
     return (
       <ReactModal
+        id={"test-id"}
```
Comment (FiberJW) [inline types/react-modal/react-modal-tests.tsx]: Thanks for your contribution!

It's a small nit, but like most of the code I read and write (including this very file), the authors forgo the curly braces when assigning a (non-template) string value as a prop to a react component. We should also do that here for consistency within the file.

Also, adding the value `"test-id"` for the `id` prop when we already have a `testId` prop seems kind of backward. Maybe we can swap those two values?

Thanks again!

```suggestion
        id="test-id"
```

**Review by FiberJW — CHANGES_REQUESTED** [review]

File: types/react-modal/react-modal-tests.tsx (hunk 1 above)
Comment (nitzanmo) [reply types/react-modal/react-modal-tests.tsx]: Hey, thanks for your review :)

I removed the curly braces and changed id's value to "modal-id"

👍

**Review by FiberJW — APPROVED** [review]
LGTM! Thanks for the contribution :) 

thumbs_up_5

**Comment by PranavSenthilnathan** [comment]
Congratulations on your first DefinitelyTyped contribution! Thank you for being a part of the community!

**Pull request merged**